content/docs/vmclarity/getting-started/deploy-kubernetes/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Deploy on Kubernetes
+weight: 110
+---
+
+## Prerequisites
+
+* Install a tool to run local Kubernetes clusters. Here, [Kind](https://kind.sigs.k8s.io/) is used as the default option for creating a local cluster.
+* Install a package manager. Here, [Helm](https://helm.sh/) is used as the default installer.
+
+## Deployment steps
+
+To deploy VMClarity to your Kubernetes cluster, complete the following steps.
+
+1. Ensure the Kubernetes cluster is up and running. If you're using kind, you can check the status of your clusters with the following command:
+
+    ```shell
+    kind get clusters
+    ````
+
+1. Use Helm to install VMClarity. Run the following command, making sure to replace installation/kubernetes/helm/vmclarity with the correct path to the VMClarity Helm chart:
+
+    ```shell
+    helm install vmclarity installation/kubernetes/helm/vmclarity \
```

Comment:
We should build and distribute the chart as an OCI chart for example, instead of pointing to the chart directory in the repo.